dev_suites/dev_demo_ig_stu1/demo_suite.rb

```diff
@@ -299,5 +300,47 @@ class DemoSuite < Inferno::TestSuite
         end
       end
     end
+
+    group do
+      title 'Custom Suite Endpoints'
+      description %(
+        This group demonstrates custom suite endpoint functionality.
+      )
+
+      input :custom_bearer_token,
+            description: 'This bearer token will be used to identify the incoming request'
+
+      suite_endpoint :post, '/suite_endpoint', DemoEndpoint
```

Comment:
i'm having trouble wrapping my head around the purpose of all different spots we need to write code.  You declare the endpoint globally (well, within the suites namespace, though its not 'in' the suite), though that endpoint doesn't include the definition of the endpoint URL or method its responds to.  That is defined in group metadata that 'activate' or 'hook into' or 'listen in on' that "endpoint"? 

Then, (optionally?), we have a run block which has a 'wait' element which is informs inferno (and the user) that this specific endpoint needs to be visited before moving on.  

Compare this to something like webmock, where it just